src/pages/tools-resources.tsx

```diff
@@ -1,7 +1,133 @@
+                title={titleEntries[0].title}
+                description={titleEntries[0].description}
+              />
+              <div className="grid grid-cols-3 h-2/3 p-4 gap-4">
+                <div className="col-span-2">
+                  <MultidiscilnaryByPracticeApproach
+                    approachEntries={multidisciplinaryByPracticeEntries}
+                  />
+                </div>
+                <div className="grid">
+                  <MultidiscilnaryByTheoryApproach
+                    approachEntries={multidisciplinaryByTheoryEntries}
+                  />
+                </div>
+              </div>
+            </div>
+
+            <div className="pt-10">
+              <div className="border-4 border-dashed border-gray-400 bg-gray-50 py-4 px-2 rounded w-2/3">
+                <TitleCard
+                  title={titleEntries[1].title}
+                  description={titleEntries[1].description}
+                />
+                <div className="grid grid-cols-2 p-3 gap-4 h-1/3">
+                  <div className="">
+                    <ParticipatoryByPracticeApproch
+                      approachEntries={participatoryApproachEntries}
+                    />
+                  </div>
+                  <div className="p-2">
+                    <ParticipatoryByTheoryApproch
+                      approachEntries={participatoryApproachEntries1}
+                    />
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div className="flex gap-10 w-1/2 -mt-[28rem]">
+        <div className="border-4 border-dashed border-gray-400 bg-gray-50 py-4 px-2 rounded">
+          <TitleCard
+            title={titleEntries[2].title}
+            description={titleEntries[2].description}
+          />
+          <div className="flex gap-3 p-4 h-fit">
+            <div className="p-3">
+              <BehaviorByPractice
+                approachEntries={behaviorByPracticeApproach}
+              />
+            </div>
+            <div className="p-3">
+              <BehaviorByTheory approachEntries={behaviorByTheoryApproach} />
+            </div>
+          </div>
+        </div>
+
+        <div className="border-4 border-dashed border-gray-400 bg-gray-50 h-fit mt-20 py-4 px-2 rounded">
+          <TitleCard
+            title={titleEntries[3].title}
+            description={titleEntries[3].description}
+          />
+          <div className="grid grid-cols-2 gap-3 p-4 h-2/3">
+            <div className="p-1">
+              <HumanDesignByPracticeApproach
+                approachEntries={humanDesignByPracticeApproach}
+              />
+            </div>
+            <div className="grid p-1">
+              <HumanDesignByTheoryApproach
+                approachEntries={humanDesignByTheoryApproach}
+              />
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  );
```

Comment:
Can you create the index file inside each section folder where you can put all of those subsections instead of putting them together here.